src/utils/env.js

```diff
@@ -0,0 +1,3 @@
+export const AGENT_LABEL = process.env.AGENT_LABEL || 'holder'
+export const CAA_ENDPOINT = process.env.CAA_ENDPOINT || 'http://localhost:8051'
+export const AGENT_ORIGIN = process.env.CAA_ENDPOINT || 'http://localhost:8031'
```

Comment:
should these be REACT_APP_? This has CAA_ENDPOINT (maybe could be AUTHORITY_ to make more generic) again instead of AGENT_ORIGIN